.github/ISSUE_TEMPLATE.md

```diff
@@ -1,41 +1,41 @@
+<!--
 ## Before you submit an issue
 Please look up our issues, maybe your problem was mentioned before.
+-->
 
 ## Description
-Provide a short description of your problem here.
+<!-- Provide a short description of your problem here. -->
 
 If you executed a specific command:
 ```shell
-your command here
+# your command here
 ```
 
 Output:
-```shell
-output
-of
-your
-command
-here
+```
+output of your command here
```

Comment:
Does indenting instead of backticks work here?

Reply:
It is easier for the user to paste the output without indenting each line

Reply:
For me personally, it's easier with the backticks.